account_financial_report/static/src/css/report_html.css

```
@@ -0,0 +1,7 @@
+/* Styles of html report */
+.account_title span {
+    font-size: 1rem;
+}
+.act_as_cell {
+    font-size: 0.9rem;
```

```suggestion
    .account_title span {
        font-size: 1rem !important;
    }
    .act_as_cell {
        font-size: 0.9rem !important;
```


> Trial balance is still using the old font size it seems.

It has a direct style so it takes precedence over the style applied to the web template, you could include a ‘!important’ in the css of the web template.

https://github.com/OCA/account-financial-reporting/blob/15.0/account_financial_report/report/templates/trial_balance.xml#L48